src/inc/msquic_posix.h

```diff
@@ -80,86 +80,60 @@ inline ENUMTYPE &operator ^= (ENUMTYPE &a, ENUMTYPE b) throw() { return (ENUMTYP
-#define ERROR_VER_NEG_ERROR              14 + ERROR_BASE    // 0xBEBC20E
-#define ERROR_EPOLL_ERROR                15 + ERROR_BASE    // 0xBEBC20F
-#define ERROR_DNS_RESOLUTION_ERROR       16 + ERROR_BASE    // 0xBEBC210
-#define ERROR_SOCKET_ERROR               17 + ERROR_BASE    // 0xBEBC211
-#define ERROR_SSL_ERROR                  18 + ERROR_BASE    // 0xBEBC212
-#define ERROR_USER_CANCELED              19 + ERROR_BASE    // 0xBEBC213
-#define ERROR_ALPN_NEG_FAILURE           20 + ERROR_BASE    // 0xBEBC214
-#define ERROR_STREAM_LIMIT_REACHED       21 + ERROR_BASE    // 0xBEBC215
-
-#define TLS_ERROR_BASE                  256 + ERROR_BASE    // 0xBEBC300
-#define CERT_ERROR_BASE                 512 + ERROR_BASE    // 0xBEBC400
-
-#define QUIC_STATUS_SUCCESS                 ((QUIC_STATUS)ERROR_SUCCESS)
-#define QUIC_STATUS_PENDING                 ((QUIC_STATUS)ERROR_NOT_READY)
-#define QUIC_STATUS_CONTINUE                ((QUIC_STATUS)ERROR_CONTINUE)
-#define QUIC_STATUS_OUT_OF_MEMORY           ((QUIC_STATUS)ENOMEM)
-#define QUIC_STATUS_INVALID_PARAMETER       ((QUIC_STATUS)EINVAL)
-#define QUIC_STATUS_INVALID_STATE           ((QUIC_STATUS)ERROR_INVALID_STATE)
-#define QUIC_STATUS_NOT_SUPPORTED           ((QUIC_STATUS)EOPNOTSUPP)
-#define QUIC_STATUS_NOT_FOUND               ((QUIC_STATUS)ENOENT)
-#define QUIC_STATUS_BUFFER_TOO_SMALL        ((QUIC_STATUS)EOVERFLOW)
-#define QUIC_STATUS_HANDSHAKE_FAILURE       ((QUIC_STATUS)ERROR_HANDSHAKE_FAILURE)
-#define QUIC_STATUS_ABORTED                 ((QUIC_STATUS)ERROR_OPERATION_ABORTED)
-#define QUIC_STATUS_ADDRESS_IN_USE          ((QUIC_STATUS)EADDRINUSE)
-#define QUIC_STATUS_CONNECTION_TIMEOUT      ((QUIC_STATUS)ETIMEDOUT)
-#define QUIC_STATUS_CONNECTION_IDLE         ((QUIC_STATUS)ERROR_CONNECTION_IDLE)
-#define QUIC_STATUS_INTERNAL_ERROR          ((QUIC_STATUS)ERROR_INTERNAL_ERROR)
-#define QUIC_STATUS_CONNECTION_REFUSED      ((QUIC_STATUS)ERROR_CONNECTION_REFUSED)
-#define QUIC_STATUS_PROTOCOL_ERROR          ((QUIC_STATUS)ERROR_PROTOCOL_ERROR)
-#define QUIC_STATUS_VER_NEG_ERROR           ((QUIC_STATUS)ERROR_VER_NEG_ERROR)
-#define QUIC_STATUS_UNREACHABLE             ((QUIC_STATUS)EHOSTUNREACH)
-#define QUIC_STATUS_PERMISSION_DENIED       ((QUIC_STATUS)EPERM)
-#define QUIC_STATUS_EPOLL_ERROR             ((QUIC_STATUS)ERROR_EPOLL_ERROR)
-#define QUIC_STATUS_DNS_RESOLUTION_ERROR    ((QUIC_STATUS)ERROR_DNS_RESOLUTION_ERROR)
-#define QUIC_STATUS_SOCKET_ERROR            ((QUIC_STATUS)ERROR_SOCKET_ERROR)
-#define QUIC_STATUS_TLS_ERROR               ((QUIC_STATUS)ERROR_SSL_ERROR)
-#define QUIC_STATUS_USER_CANCELED           ((QUIC_STATUS)ERROR_USER_CANCELED)
-#define QUIC_STATUS_ALPN_NEG_FAILURE        ((QUIC_STATUS)ERROR_ALPN_NEG_FAILURE)
-#define QUIC_STATUS_STREAM_LIMIT_REACHED    ((QUIC_STATUS)ERROR_STREAM_LIMIT_REACHED)
+#ifndef ESTRPIPE // undefined on macOS it seems
+#define ESTRPIPE 86
+#endif // ESTRPIPE
+
+#define ERROR_BASE                          200000000                       // 0xBEBC200
```

Comment:
To be "outside the range of all possible `int` values" shouldn't this be negative?

Reply:
I'm just going to leave as is.